x-pack/plugins/security_solution/cypress/tasks/api_calls/prebuilt_rules.ts

```diff
@@ -109,6 +115,48 @@ export const waitUntilAllRuleAssetsCreated = (
     { interval: 500, timeout: 12000 }
   );
 
+/*
+ * Wait until all rules assets passed in as parameters are
+ * created in the `.kibana_alerting_cases` index as actual installed rules
+ * (saved objects of type alert)
+ */
+export const waitUntilAllRulesCreated = (
```

Comment:
It looks like there is another way to wait for rule installation inside `installAllPrebuiltRulesRequest()`. The only problem it doesn't know which rules to wait for so it can fetch this list. If implement it this way it will help to simplify the tests.